File: sort.py
import random


def insertionSort(arr: list):
    for i in range(1, len(arr)):
        current = arr[i]
        j = i - 1

        while j >= 0 and arr[j] > current:
            arr[j + 1] = arr[j]
            j -= 1

        arr[j + 1] = current


res = [random.randrange(1, 1000, 1) for _ in range(10)]
print(res)
insertionSort(res)
print(res)

File: sort.js
function insertionSort(arr) {
  for (let i = 1; i < arr.length; i++) {
    let current = arr[i];
    let j = i - 1;

    while (j >= 0 && arr[j] > current) {
      arr[j + 1] = arr[j];
      j--;
    }

    arr[j + 1] = current;
  }
}

let data = [];
for (let i = 0; i < 10; i++) {
  data.push(Math.floor(Math.random() * 1000));
}

console.log(data);
insertionSort(data);
console.log(data);
